packages/twenty-front/src/modules/object-record/object-filter-dropdown/constants/SelectFilterTypes.ts

```diff
@@ -0,0 +1 @@
+export const SELECT_FILTER_TYPES = ['SELECT', 'MULTI_SELECT'];
```

Comment:
this seems to be only used in ViewBarFilterEffect, I'm not sure it makes sense to introduce a typing for that. It does not make a lot of sense to me